keps/sig-apps/2599-minreadyseconds-for-statefulsets/README.md

```diff
@@ -603,11 +619,23 @@ For each of them, fill in the following information by copying the below templat
       Not required until feature graduated to beta.
     - Testing: Are there any tests for failure mode? If not, describe why.
 -->
+ - `minReadySeconds` not respected and all the pods are shown `Available` immediately
```

Comment:
good answer.

I think this one will be caught by users and the next one could/should be caught by cluster-admins if it is systemic.